BLSQ/iaso
# IA-2786: Add data download (CSV/Excel) blurred buttons on the submissions page

Explain what problem this PR is resolving

Related JIRA tickets : [IA-2786](https://bluesquare.atlassian.net/browse/IA-2786)

## Self proofreading checklist


## Changes

Added  **Tooltip**  on search button to explain what to do before search submissions!

## How to test

Mouse over on the tooltip next the search button on the submissions page and see the appearing text.

## Print screen / video

https://github.com/BLSQ/iaso/assets/6383219/59247940-2ac4-4384-a1dc-dc36cd3f2989





[IA-2786]: https://bluesquare.atlassian.net/browse/IA-2786?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
